OrbitGl/TimeGraph.cpp

```diff
@@ -421,6 +441,58 @@ void TimeGraph::SelectRight(const TextBox* a_TextBox) {
   SetMinMax(minTimeUs, maxTimeUs);
 }
 
+//-----------------------------------------------------------------------------
+TextBox* TimeGraph::GetPrevious(uint64_t function_address,
```

Comment:
Instead of "Get" I would prefer "Find" because it implies that there is some work to do and that the result might be not found (nullptr). Also I would specify what will be returned, maybe something like `FindPreviousFunctionCall`? Sorry I don't have a suggestion for a better name.

Reply:
Done!